src/zip.ts

```diff
@@ -88,6 +96,16 @@ export const zipFunctions = async function (
         srcPath: func.srcPath,
         stat: func.stat,
       })
+      const durationNs = endTimer(startIntervalTime)
+      const durationMs = roundTimerToMillisecs(durationNs)
+      const logObject = {
+        name: func.name,
+        config: func.config,
+        featureFlags: functionFlags,
+        durationMs,
```

Comment:
Is there any reason for us not to use nanoseconds here? Feels a bit strange that we're reaching for the high-resolution time for the extra precision and then losing it when emitting the log.

Reply:
I'm thinking during an investigation, what would be faster for us to compute/make sense of at a glance? i.e would it be  easier to see 600000ms as 10min or 600000000000ns as 10min? Initial issue [request](https://github.com/netlify/pod-compute/issues/246#issue-1420815403) was also for milliseconds. 

The conversion from nanoseconds is because the [process.hrtime](https://nodejs.org/api/process.html#processhrtimetime) operation gives us time in nanoseconds.

Reply:
We're talking about the duration of bundling individual functions here, not about the entire bundling step. So 10 minutes is really not the order of magnitude we should be dealing with. Small functions should be processed really quickly, and we might actually be losing precision by converting to milliseconds.

Reply:
Okay, PR updated.